site/docs/operations/docker.md

```diff
@@ -0,0 +1,340 @@
+```
+
+You should eventually see a 3 node cluster formed, note that you will
+see the output multiple times, prefixed by the container name:
+
+```text
+jet_3  | Members {size:3, ver:3} [
+jet_3  |    Member [172.21.0.3]:5701 - 99d3de67-8c5d-452b-8165 -085a4cd1fcda
+jet_3  |    Member [172.21.0.2]:5701 - 64f5b01b-847e-49e0-87f2 -db2a6f7750b7
+jet_3  |    Member [172.21.0.4]:5701 - ffa362f9-617d-42bc-a74c -05ce857e8e48 this
+jet_3  | ]
+```
+
+The `ports` section says that port 5701 from each container should be
+mapped to a port from randge 5701-5703. Increase the range if you would
+like to start more than 3 instances.
+
+### Configuration
+
+You can provide custom `hazelcast.yaml` or `hazelcast-jet.yaml`
+configuration files by using a volume:
+
+```yaml
+version: '3'
+
+services:
+
+  jet:
+    image: hazelcast/hazelcast-jet
+    ports:
+      - "5701-5703:5701"
+    volumes:
+      - ./hazelcast.yaml:/opt/hazelcast-jet/config/hazelcast.yaml
+```
+
+To see the default content of the files run the following command:
+
+```bash
+docker run hazelcast/hazelcast-jet cat /opt/hazelcast-jet/config/hazelcast-jet.yaml
+```
+
+## Package job as a docker image
+
+When your architecture is Docker focused it might be easier to have
+also your job packaged as a Docker image. We will use Hazelcast Jet
+image as base image and create an image with hello world job. You can
+use the same approach to package any job.
+
+Create a file called `Dockerfile` in the <jet home> directory.
+
+```text
+FROM hazelcast/hazelcast-jet
+
+ADD examples/hello-world.jar /examples/
+
+ENV ADDRESSES 172.17.0.2
+
+CMD ["sh", "-c", "jet -a $ADDRESSES submit /examples/hello-world.jar"]
+```
+
+The first line specifies `hazelcast/hazelcast-jet` image as base image.
+The `ADD` command copies the `hello-world.jar` to the `/examples`
+directory inside the container image. `ENV` command defines an
+environment variable `ADDRESSES` with default value of `172.17.0.2`.
+This can be overriden via `-e` parameter of the `docker run`
+command. The `CMD` command specifies what command to run.
+
+Then create the image by running the following command:
+
+```bash
+docker build . -t hazelcast-jet-hello-world
+```
+
+You should see output similar to this:
+
+```text
+Sending build context to Docker daemon  77.35MB
+Step 1/4 : FROM hazelcast/hazelcast-jet:{jet-version}
+ ---> c126c24c3d0b
+Step 2/4 : ADD examples/hello-world.jar /examples/
+ ---> Using cache
+ ---> 2e3fa98d0c7e
+Step 3/4 : ENV ADDRESSES 172.17.0.2
+ ---> Using cache
+ ---> f96d01158d6e
+Step 4/4 : CMD ["sh", "-c", "jet -a $ADDRESSES submit /examples/hello-world.jar"]
+ ---> Using cache
+ ---> 6bc0f527b69c
+Successfully built 6bc0f527b69c
+Successfully tagged hazelcast-jet-hello-world:latest
+```
+
+Now you can submit the job by creating a container from the newly built
+image:
+
+```bash
+docker run -it hazelcast-jet-hello-world
+```
+
+> You don't need the `-it` parameters if your job class exits after
```

Comment:
... parameter ...

Reply:
It's actually 2 separate parameters here, could be written as `-i -t`, but commonly as `-it`.